src/gromacs/nblib/tests/util.cpp

```diff
@@ -91,27 +94,15 @@ TEST(NBlibTest, checkNumericValuesHasInf)
 }
 
 
-TEST(NBlibTest, generateVelocity)
+TEST(NBlibTest, generatedVelocitiesAreCorrect)
```

Comment:
This is a typo. It has to be `GeneratedVelocitiesAreCorrect` with a capital G for the tests to pass.